admin/client/App/store.js

```diff
@@ -46,9 +46,10 @@ store.subscribe(() => {
 	// Get all the necessary data
 	const state = store.getState();
 	const list = state.lists.currentList;
-	const items = state.lists.items;
+	// const items = state.lists.items;
```

Comment:
nit: any reason for this to stay here? If so, add a comment explaining it please